Dockerfile

```diff
@@ -1,110 +1,85 @@
-RUN apt-get clean && rm -rf /var/lib/apt/lists/*
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-    tzdata 
-RUN apt-get update && apt-get install -y --no-install-recommends  \
-    bzip2                 \
-    build-essential       \
+RUN apt-get update && apt-get install -y  \
     ca-certificates       \
     cp2k                  \
     file                  \
-    git                   \
-    gnupg                 \
-    graphviz              \
-    locales               \
-    less                  \
     libssl-dev            \
     libffi-dev            \
-    psmisc                \
-    python-dev            \
-    python-pip            \
-    python-setuptools     \
-    python-wheel          \
-    python3-pip           \
-    python3-setuptools    \
-    python3-wheel         \
-    python-tk             \
     quantum-espresso      \
-    rsync                 \
-    ssh                   \
-    unzip                 \
-    vim                   \
-    wget                  \
-    zip                   \
   && rm -rf /var/lib/apt/lists/*
 
-# Add repo for postgres-9.6
-RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ bionic-pgdg main" >> /etc/apt/sources.list.d/pgdg.list
-RUN wget -q https://www.postgresql.org/media/keys/ACCC4CF8.asc -O - | apt-key add -
-RUN apt-get update && apt-get install -y --no-install-recommends  \
-    postgresql-9.6        \
+# needed for jupyterlab
+RUN apt-get update && apt-get install -y \
+     nodejs                \
+     npm                   \
   && rm -rf /var/lib/apt/lists/*
 
-# fix locales
-RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && locale-gen
-ENV LC_ALL en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US.UTF-8
-
 
 # Quantum-Espresso Pseudo Potentials
 WORKDIR /opt/pseudos
 RUN base_url=http://archive.materialscloud.org/file/2018.0001/v2;  \
     for name in SSSP_efficiency_pseudos SSSP_precision_pseudos; do \
-       wget ${base_url}/${name}.aiida;                             \
+       wget ${base_url}/${name}.tar.gz;                            \
+       tar -zxvf ${name}.tar.gz;                                   \
     done;                                                          \
     chown -R root:root /opt/pseudos/;                              \
     chmod -R +r /opt/pseudos/
 
-## install rclone
-#WORKDIR /opt/rclone
-#RUN wget https://downloads.rclone.org/rclone-v1.38-linux-amd64.zip;  \
-#    unzip rclone-v1.38-linux-amd64.zip;                              \
-#    ln -s rclone-v1.38-linux-amd64/rclone .
-
-# install PyPI packages for Python 3
+# install packages that are not in the aiidalab meta package
+# 'fastentrypoints' is to fix problems with aiida-quantumespresso plugin installation
 RUN pip3 install --upgrade         \
-    'tornado==5.0.2'               \
+    'fastentrypoints'              \
+    'tornado==5.1.1'               \
     'jupyterhub==0.9.4'            \
```

Comment:
no upgrade to 1.0 yet?

Reply:
I am not sure if we can do it before resolving tornado issue in AiiDA core. I will test it and make a separate commit if it will works.